lib/seattleflu/db/cli/command/etl/__init__.py

```diff
@@ -140,4 +142,106 @@ def upsert_encounter(db: DatabaseSession,
     return encounter
 
 
+def sample_identifier(db: DatabaseSession, barcode: str) -> str:
+    """
+    Find corresponding UUID for scanned sample or collection barcode within
+    warehouse.identifier.
+
+    Will be sample barcode if from UW and collection barcode if from SCH.
+    """
+    identifier = find_identifier(db, barcode)
+
+    if identifier:
+        assert identifier.set_name == "samples" or \
+            identifier.set_name == "collections-seattleflu.org", \
+            f"Identifier found in set «{identifier.set_name}», not «samples»"
+
+    return str(identifier.uuid) if identifier else None
+
+
+def find_sample(db: DatabaseSession, identifier: str) -> Any:
+    """
+    Find sample by *identifier* and return sample.
+    """
+    LOG.debug(f"Looking up sample «{identifier}»")
+
+    sample = db.fetch_row("""
+        select sample_id as id, identifier, encounter_id
+          from warehouse.sample
+         where identifier = %s or
+               collection_identifier = %s
+           for update
+        """, (identifier,identifier,))
+
+    if not sample:
+        LOG.error(f"No sample with identifier «{identifier}» found")
+        return None
+
+    LOG.info(f"Found sample {sample.id} «{sample.identifier}»")
+    return sample
+
+
+def update_sample(db: DatabaseSession,
```

Comment:
Ah, I totally forgot we had this function in `etl clinical`, I wrote another one very similar in https://github.com/seattleflu/id3c/pull/35

Reply:
There's certainly a lot to keep track of, and in the future we may want something better than shoving everything into `__init__.py`. I can refactor the function from #35 if that PR gets merged to master before this one.

Reply:
We definitely want something better in the future, but for now, let's leave these functions in `__init__.py`.